demos/keyboard-a11y/js/five-star.js

```diff
@@ -11,194 +11,193 @@ You may obtain a copy of the ECL 2.0 License and BSD License at
 https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
 */
 
-var demo = demo || {};
-
-(function ($, fluid) {
-    "use strict";
+"use strict";
 
-    fluid.registerNamespace("demo.fiveStar");
+var demo = demo || {};
```

Comment:
Out of curiosity, why isn't enough to just call `fluid.registerNamespace("demo.fiveStar");`

Reply:
Oh I guess because of the function declarations.